src/components/ListItem/ListItem.stories.js

```diff
@@ -43,6 +43,9 @@ const defaultExample = () => ({
     hasImage: {
       default: boolean('Show image?', false),
     },
+    itemValue: {
+      default: text('Value (If any)', ''),
```

Comment:
After renaming `itemValue` to: `rightLabel`, please update the `text` label here to: `Right label`